src/adagrams.js

```diff
@@ -1,15 +1,200 @@
+const LETTER_POOL = [
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "A",
+  "B",
+  "B",
+  "C",
+  "C",
+  "D",
+  "D",
+  "D",
+  "D",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "E",
+  "F",
+  "F",
+  "G",
+  "G",
+  "G",
+  "H",
+  "H",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "I",
+  "J",
+  "K",
+  "L",
+  "L",
+  "L",
+  "L",
+  "M",
+  "M",
+  "N",
+  "N",
+  "N",
+  "N",
+  "N",
+  "N",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "O",
+  "P",
+  "P",
+  "Q",
+  "R",
+  "R",
+  "R",
+  "R",
+  "R",
+  "R",
+  "S",
+  "S",
+  "S",
+  "S",
+  "T",
+  "T",
+  "T",
+  "T",
+  "T",
+  "T",
+  "U",
+  "U",
+  "U",
+  "U",
+  "V",
+  "V",
+  "W",
+  "W",
+  "X",
+  "Y",
+  "Y",
+  "Z",
+];
+
+const LETTER_VALUES = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10,
+};
+
+const LONG_WORD_BONUS = 8;
+const LONG_WORD_RANGE = [7, 8, 9, 10];
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  // shuffle LETTER_POOL array
+  const shuffled = LETTER_POOL.sort(() => 0.5 - Math.random());
+  // slice the first 10 letters of shuffled array to create hand
+  let hand = shuffled.slice(0, 10);
+  return hand;
```

Comment:
It looks like this mutates the letter pool. That means if we run it a few times, we'll eventually run out of letters! Consider making a copy of the letter pool before modifying it.